cluster_pack/skein/yarn_launcher.py

```diff
@@ -108,18 +108,32 @@ def _submit(
         max_restarts=max_restarts
     )
 
-    spec = skein.ApplicationSpec(
-        name=name,
-        file_systems=hadoop_file_systems,
-        services={name: service},
-        acls=skein.model.ACLs(
-            enable=True,
-            ui_users=['*'],
-            view_users=['*']
-        ),
-        # acquire_map_reduce_delegation_token=acquire_map_reduce_delegation_token,
-        max_attempts=max_attempts
-    )
+    # workaround for https://github.com/jcrist/skein/pull/197
+    if hasattr(skein.ApplicationSpec, 'acquire_map_reduce_delegation_token'):
+        spec = skein.ApplicationSpec(
+            name=name,
+            file_systems=hadoop_file_systems,
+            services={name: service},
+            acls=skein.model.ACLs(
+                enable=True,
+                ui_users=['*'],
+                view_users=['*']
+            ),
+            acquire_map_reduce_delegation_token=acquire_map_reduce_delegation_token,
+            max_attempts=max_attempts
+        )
+    else:
+        spec = skein.ApplicationSpec(
+            name=name,
+            file_systems=hadoop_file_systems,
+            services={name: service},
+            acls=skein.model.ACLs(
+                enable=True,
+                ui_users=['*'],
+                view_users=['*']
+            ),
+            max_attempts=max_attempts
+        )
```

Comment:
It would be better to instantiate spec in a first time, then:
`if hasattr(skein.ApplicationSpec, 'acquire_map_reduce_delegation_token'):
    spec.acquire_map_reduce_delegation_token = acquire_map_reduce_delegation_token`

Reply:
Done